Dockerfile-edge

```diff
@@ -95,68 +98,29 @@ RUN apt-get update -q && \
     # Install new PHP7-stable version of Yaml \
     pecl install yaml-2.0.0 && \
     echo "extension=yaml.so" > $CONF_PHPMODS/yaml.ini && \
-    # Install new PHP7-stable version of Redis \
+    # Install new PHP7-stable version of Redis
     pecl install redis-3.0.0 && \
```

Comment:
its not being used by default, but it is available for teams to use, which do exist today